src/System.Management.Automation/engine/remoting/fanin/WSManPlugin.cs

```diff
@@ -1,7 +1,7 @@
 // ----------------------------------------------------------------------
 //
 //  Microsoft Windows NT
-//  Copyright (C) Microsoft Corporation, 2007.
+//  Copyright (c) Microsoft Corporation, 2007.
```

Comment:
I believe we use common pattern "Copyright (c) Microsoft.  All rights reserved."

Below too.